test/integ/cli/test_cli_manifest_snapshot.py

```diff
@@ -14,8 +14,7 @@
 from deadline.client.cli import main
 
 
-@pytest.mark.skip("Random Failure with no credentials on Github")
-class TestSnapshot:
```

Comment:
Curious, what was the reason and what has been done to avoid the random failures?

Reply:
This file was moved from unit tests. In Unit tests, the mocks changed the AWS creds, and that caused a very weird behavior depending on how unit tests were scheduled. This caused the credentials to be suddenly unmocked!